### DISCUSSION TITLE
Is it possible to fit the image in a page from the random link?

### DISCUSSION THREAD

**Discussion by gerroon** [bpatrik/pigallery2#640] [Q&A]
Hi

I use the random link option frequently, it is great but if the image is large the image is not fitting in the page (the image is shown at %100 and requires scrolling). Is there a way to force the image to fit in the current browser window at all?

thanks

**Comment by bpatrik** [comment]
Random link shows you the raw image. There is no html around it. Your
device decides how to show it.

see #426
